code/game/objects/buckling.dm

```diff
@@ -1,103 +1,118 @@
 /obj
 	var/can_buckle = 0
+	var/can_buckle_bags = 0
```

Comment:
This mechanic shouldn't be specific to bags if implemented. Objects should be able to decide what exactly can be buckled to them via either a list or a proc that returns a list of objects.

Reply:
I knew I missed something in my clean-up. I meant to entirely unify the buckling into one thing, but obviously missed this.